dotnet-desktop-guide/net/winforms/data/overview.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Overview of Data Binding and Windows Forms"
+description: "Learn about interfaces related to Data Binding, data sources supported by Windows Forms, and types of Data Binding."
+ms.date: "05/05/2022"
+ms.topic: overview
+helpviewer_keywords:
+  - "data [Windows Forms], data-binding interfaces"
+  - "INotifyPropertyChanged interface"
+  - "IBindingListView interface"
+  - "IList interface [Windows Forms], Windows Forms data binding"
+  - "IBindingList interface [Windows Forms], Windows Forms data binding"
+  - "interfaces [Windows Forms], Windows Forms data binding"
+  - "IEditableObject interface [Windows Forms], Windows Forms data binding"
+  - "data binding [Windows Forms], interfaces"
+  - "IDataErrorInfo interface [Windows Forms], Windows Forms data binding"
+ms.custom: devdivchpfy22
+---
+
+# Data Binding and Windows Forms
+
+In Windows Forms, you can bind to not just traditional data sources, but also to almost any structure that contains data. You can bind to an array of values that you calculate at run time, read from a file, or derive from the values of other controls.
+
+ In addition, you can bind any property of any control to the data source. In traditional data binding, you typically bind the display property—for example, the <xref:System.Windows.Forms.Control.Text%2A> property of a <xref:System.Windows.Forms.TextBox> control—to the data source. With the .NET, you also have the option of setting other properties through binding as well. You might use binding to perform the following tasks:
```

Comment:
With the .NET, you also have the option of setting other properties through binding as well. --> Remove as well, you have already used "also" in the sentence earlier.